### PR TITLE
Improve REAMDE documentation

### LINKED ISSUE #34: Missing information in README
Extend the readme.md file with further information about the WAM_APP_FRED. 

- What is the purpose of this app
- How does this app work/ how is it used
- What are its functionalitys
- Explain the source data/ where can it be found 
- How can the data be accessed

### PR DESCRIPTION
**Pull request by jh-RLI**
closes #34

### CODE REVIEW

**Comment by jh-RLI** [comment]
Must be reviewed @Bachibouzouk  @christian-rli .

File: README.md [hunk 1]
```diff
@@ -1,8 +1,21 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easy assessable and platform-independent web-application that can be used via the
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
to create an easily accessible and platform-independent web-application that can be used via the 
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,21 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The WAM_APP_FRED is developed as an open-source web-application to visualize data that was collected or created 
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,21 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easy assessable and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv). By that an user is able to explore
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
also provides functionality to get the data in a non proprietary format (like .csv).
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,21 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easy assessable and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv). By that an user is able to explore 
+the data in order to get a better understanding about the data.
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion

```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,21 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easy assessable and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv). By that an user is able to explore 
+the data in order to get a better understanding about the data. 
+To make the purpose of what is visualized on the map or as graph more oblivious the source data is divided into
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The source data is divided into 3 main categories :
```

File: README.md [hunk 6]
```diff
@@ -1,8 +1,21 @@
...
 
+The WAM_APP_FRED is developed as an open-source web-application which visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easy assessable and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv). By that an user is able to explore 
+the data in order to get a better understanding about the data. 
+To make the purpose of what is visualized on the map or as graph more oblivious the source data is divided into 
+three main topics. These three topics are: Weather-Data, powerplant asset register and feedin-time-series. Each of these
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
: Weather-Data, powerplant asset register and feedin-time-series. Each of these 
```

File: README.md [hunk 7]
```diff
@@ -1,8 +1,21 @@
...
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv). By that an user is able to explore 
+the data in order to get a better understanding about the data. 
+To make the purpose of what is visualized on the map or as graph more oblivious the source data is divided into 
+three main topics. These three topics are: Weather-Data, powerplant asset register and feedin-time-series. Each of these 
+topics basically have there own source data. The data is provided within the open-energy-platform. Further explanation 
+is given in the section source data. 
+
+To achieve these requirements we use proven open-source technologies. We use django as Web-Framework. The base structure
```
Comment (Bachibouzouk) [inline README.md]: what does "proven open-source" mean?

File: README.md [hunk 8]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+TDB
+
+### Explore feedin time series data
+
+The feedin time series data needs to be uploaded to the OEDB. Once this is finished it will be possible to present 
+the data as graph within the WAM_APP_FRED. 
+
+### Get data as .CSV file
+
+The displayed data can be downloaded as csf file. Therefore click on the button located in the control-panel
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The displayed data can be downloaded as csv file.
```

File: README.md [hunk 9]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+on the left. For example, the CSV contains all the data for a single weather point selected on the map.
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The file contains all the weather data for a single point selected on the map. 
```

File: README.md [hunk 10]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+
+The displayed data can be downloaded as csf file. Therefore click on the button located in the control-panel
+on the left. For example, the CSV contains all the data for a single weather point selected on the map. 
+
+## Source database
+
+### Introducing the Open-Energy-Database
+
+The database which is used as main data-source for all of the topics is the open-energy-database (OEDB). 
+The OEDB is a postgreSQL database that was developed within the open_eGo project.
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The OEDB is a postgreSQL database that was developed within the [open_eGo project](https://reiner-lemoine-institut.de/open_ego-open-electricity-grid-optimization/). 
```

File: README.md [hunk 11]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+### Introducing the Open-Energy-Database
+
+The database which is used as main data-source for all of the topics is the open-energy-database (OEDB). 
+The OEDB is a postgreSQL database that was developed within the open_eGo project. 
+
+### Introducing the Open-Energy-Platform
+
+The OEDB is managed within the openenergy-platform (OEP). The OEP is the presentation layer for the OEDB. The OEP 
+provides the Open-Energy Dialect (OED) which is a RestFULL API. Using the OED one can query the OEDB. The OEP provides 
+the open-energy-dialect (OED) which is the main RestFull-API to query the OEDB. The data should be provided as JSON or
```
Comment (Bachibouzouk) [inline README.md]: should or must?

File: README.md [hunk 12]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+### Introducing the Open-Energy-Platform
+
+The OEDB is managed within the openenergy-platform (OEP). The OEP is the presentation layer for the OEDB. The OEP 
+provides the Open-Energy Dialect (OED) which is a RestFULL API. Using the OED one can query the OEDB. The OEP provides 
+the open-energy-dialect (OED) which is the main RestFull-API to query the OEDB. The data should be provided as JSON or 
+GEOJSON. 
+
+ - OED on [GitHub](https://github.com/OpenEnergyPlatform/oedialect). 
+
+In order to use the  OED one have to provide user credential. The credentials are represented by USER-NAME
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
In order to use the OED, one have to provide user credential. The credentials are represented by USER-NAME 
```

Comment (Bachibouzouk) [reply README.md]: What do you mean by `represented by`?

File: README.md [hunk 13]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+
+- OEP can be found [here](https://openenergy-platform.org/)
+
+### Access data from OEP
+
+TBD
+
+## Source data 
+
+The following will give an overview over the data-sources used within the WAM_APP_FRED and provides where to find
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
The following is an overview of the data sources used within the WAM_APP_FRED and provides where to find 
```

File: README.md [hunk 14]
```diff
@@ -90,4 +103,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+- OEP can be found [here](https://openenergy-platform.org/)
+
+### Access data from OEP
+
+TBD
+
+## Source data 
+
+The following will give an overview over the data-sources used within the WAM_APP_FRED and provides where to find 
+and how to access them. The WAM_APP_FRED defines three main topics . Each of these topics has its own source data.
```
Comment (Bachibouzouk) [inline README.md]:
```suggestion
and how to access them.
```

**Review by Bachibouzouk — APPROVED** [review]
@jh-RLI - nice work :)

A few comments : 

- Make sure we use Django everywhere, sometimes you write django.
- Same with RESTful: it is not consistently named throughout the text.
- I would use `category` instead of `topic` :)

I already approve the PR so you don't need to re-ask my review after you're done with the changes

File: README.md (hunk 11 above)
Comment (jh-RLI) [reply README.md]: i think "must be provided as a non-proprietary format like json" is better

File: README.md (hunk 7 above)
Comment (jh-RLI) [reply README.md]: I mean  "proven and open-source"

File: README.md [hunk 15]
```diff
@@ -1,8 +1,20 @@
 # WAM_APP_FRED
 
+The WAM_APP_FRED is developed as an open-source web-application to visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easily accessible and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application
```
Comment (christian-rli) [inline README.md:7]:
```suggestion
or additionally as graph. The app provides options for the user to interact with the visualized data. The web-application 
```

File: README.md [hunk 16]
```diff
@@ -1,8 +1,20 @@
...
 
+The WAM_APP_FRED is developed as an open-source web-application to visualize data that was collected or created 
+within the project open_FRED. All data sources which are used within this web-application are open-source. The goal is 
+to create an easily accessible and platform-independent web-application that can be used via the 
+internet (using a web-browser). The main functionality the web-application provides is to visualize data on a map 
+or additionally as graph. The app provide options for the user to interact with the visualized data. The web-application 
+also provides functionality to get the data in a non proprietary format (like .csv).
+The source data is divided into 3 main categories :
+: Weather-Data, powerplant asset register and feedin-time-series. Each of these 
+topics basically have there own source data. The data is provided within the open-energy-platform. Further explanation
```
Comment (christian-rli) [inline README.md]:
```suggestion
topics basically has its own source data. The data is provided within the open-energy-platform. Further explanation 
```

File: README.md [hunk 17]
```diff
@@ -90,4 +102,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+
+The displayed data can be downloaded as csf file. Therefore click on the button located in the control-panel
+on the left. For example, the CSV contains all the data for a single weather point selected on the map. 
+
+## Source database
+
+### Introducing the Open-Energy-Database
+
+The database which is used as main data-source for all of the topics is the open-energy-database (OEDB). 
+The OEDB is a postgreSQL database that was developed within the [open_eGo project](https://reiner-lemoine-institut.de/open_ego-open-electricity-grid-optimization/).
```
Comment (christian-rli) [inline README.md:122]:
```suggestion
The OEDB is a PostgreSQL database that was developed within the [open_eGo project](https://reiner-lemoine-institut.de/open_ego-open-electricity-grid-optimization/). 
```

File: README.md [hunk 18]
```diff
@@ -90,4 +102,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+
+### Introducing the Open-Energy-Database
+
+The database which is used as main data-source for all of the topics is the open-energy-database (OEDB). 
+The OEDB is a postgreSQL database that was developed within the [open_eGo project](https://reiner-lemoine-institut.de/open_ego-open-electricity-grid-optimization/). 
+
+### Introducing the Open-Energy-Platform
+
+The OEDB is managed within the openenergy-platform (OEP). The OEP is the presentation layer for the OEDB. The OEP 
+provides the Open-Energy Dialect (OED) which is a RestFULL API. Using the OED one can query the OEDB. The OEP provides
```
Comment (christian-rli) [inline README.md]:
```suggestion
provides the Open-Energy Dialect (OED) which is a RESTful API. Using the OED one can query the OEDB. The OEP provides 
```

File: README.md (hunk 11 above)
Comment (christian-rli) [reply README.md]:
```suggestion
the open-energy-dialect (OED) which is the main RESTful API to query the OEDB. The data should be provided as JSON or 
```

File: README.md [hunk 19]
```diff
@@ -90,4 +102,68 @@ Then, click on the map and a popup with the closest weather data point informati
...
+### Introducing the Open-Energy-Platform
+
+The OEDB is managed within the openenergy-platform (OEP). The OEP is the presentation layer for the OEDB. The OEP 
+provides the Open-Energy Dialect (OED) which is a RestFULL API. Using the OED one can query the OEDB. The OEP provides 
+the open-energy-dialect (OED) which is the main RestFull-API to query the OEDB. The data should be provided as JSON or 
+GEOJSON. 
+
+ - OED on [GitHub](https://github.com/OpenEnergyPlatform/oedialect). 
+
+In order to use the OED, one have to provide user credential. The credentials are represented by USER-NAME
```
Comment (christian-rli) [inline README.md:133]:
```suggestion
In order to use the OED, one has to provide user credential. The credentials are represented by USER-NAME 
```

**Review by christian-rli — APPROVED** [review]
Thank you for extending the README! Good job. I only had a few spelling remarks.

**Comment by jh-RLI** [comment]
I will merge this pr since I updated all suggestions form your review @Bachibouzouk

**Pull request merged**